tex/tudathesis.cfg

```diff
@@ -492,7 +492,7 @@ Die Arbeit hat bisher noch nicht zu Prüfungszwecken gedient.
 \bigskip
 }{
 \section*{Erklärung zur Abschlussarbeit\\gemäß \S{}23~Abs.~7~APB der TU~Darmstadt}
-Hiermit versichere ich, , die vorliegende \TUDa@thesisType{} ohne Hilfe Dritter und nur mit den angegebenen Quellen und Hilfsmitteln angefertigt zu haben. Alle Stellen, die  Quellen entnommen wurden, sind als solche kenntlich gemacht worden. Diese Arbeit hat in gleicher oder ähnlicher Form noch keiner Prüfungsbehörde vorgelegen.
+Hiermit versichere ich, #1, die vorliegende \TUDa@thesisType{} ohne Hilfe Dritter und nur mit den angegebenen Quellen und Hilfsmitteln angefertigt zu haben. Alle Stellen, die  Quellen entnommen wurden, sind als solche kenntlich gemacht worden. Diese Arbeit hat in gleicher oder ähnlicher Form noch keiner Prüfungsbehörde vorgelegen.
```

Comment:
as mentioned on #12 this was wrong, could you implement the correction posted there?

Reply:
https://github.com/tudace/tuda_latex_templates/commit/63a4a25bed31c98cc2ff2dbb34ea52684c4c441f